packages/office-ui-fabric-react/src/components/DetailsList/__snapshots__/DetailsRow.test.tsx.snap

```diff
@@ -799,6 +803,8 @@ exports[`DetailsRow renders details list row correctly 1`] = `
                     </div>
                   </div>
                   <div
+                    aria-posinset={3}
```

Comment:
I'm a bit confused here on this one. This might be wrong, but I'm not sure. Did you test how this reads?

I would have assumed that these aria attributes should be on a focusable entity.